src/meta/processors/jobMan/MetaJobExecutor.cpp

```diff
@@ -131,6 +131,17 @@ ErrOrHosts MetaJobExecutor::getLeaderHost(GraphSpaceID space) {
     return hosts;
 }
 
+void MetaJobExecutor::interruptExecution(JobID jobId) {
+    if (jobId == jobId_) {
+        std::unique_lock<std::mutex> lk(muInterrupt_);
+        stopped_ = true;
+        condInterrupt_.notify_one();
+        lk.unlock();
```

Comment:
A question, Is this line of code redundant?

Reply:
just want to release lock , before that below LOG(INFO)